src/Sylius/Bundle/CoreBundle/Tests/Controller/ShippingCategoryControllerTest.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+namespace Sylius\Bundle\CoreBundle\Tests\Controller;
+
+use Lakion\ApiTestCase\JsonApiTestCase;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class ShippingCategoryControllerTest extends JsonApiTestCase
+{
+    public function testCreateShippingCategoryAccessDeniedResponse()
+    {
+        $this->client->request('POST', '/api/shipping-categories/');
+
+        $response = $this->client->getResponse();
+        $this->assertResponse($response, 'authentication/access_denied', 401);
+    }
+
+    public function testCreateShippingCategoryValidationFailResponse()
+    {
+        $this->resetDataBase();
+
+        $this->client->request('POST', '/api/shipping-categories/', [], [], [
+            'HTTP_Authorization' => 'Bearer access_token',
+            'CONTENT_TYPE' => 'application/json',
+        ]);
+
+        $response = $this->client->getResponse();
+        $this->assertResponse($response, 'shipping_category/create_validation_fail', 400);
+    }
+
+    public function testCreateShippingCategoryResponse()
+    {
+        $this->resetDataBase();
+
+        $this->client->request('POST', '/api/shipping-categories/', [], [], [
+            'HTTP_Authorization' => 'Bearer access_token',
+            'CONTENT_TYPE' => 'application/json',
+        ], '{"name": "Regular", "description": "Regular weight items", "code": "reg"}');
```

Comment:
Can we somehow make this call and JSON more readable here? Can we write JSON in "pretty" form?